Gemfile

```diff
@@ -1,4 +1,6 @@
 source 'http://rubygems.org'
 
-# Specify your gem's dependencies in brewdler.gemspec
-gemspec
+gem 'rspec'
```

Comment:
Probably want these to be `group :development do` or `group :test do` blocks (just for cleanliness)